composer.json

```diff
@@ -14,7 +14,9 @@
         }
     ],
     "require": {
-        "illuminate/collections": "^10.0"
+        "illuminate/collections": "^10.0",
+        "illuminate/support": "^10.1",
+        "mockery/mockery": "^1.5"
```

Comment:
This probably belongs in `require-dev`, no?

Reply:
It was automatically added here.